### PR TITLE
CFA changes...

### PR DESCRIPTION
**Pull request by nikk1e**
Key unlock page.  Will show if you comment out passphrase property on fixtures/keys.js.  Holds until server restarts.  Need to add a KillSwitch API to "forget" the passphrase and force a page refresh.

Messages en/decrypt when going to/from mongo.  If they dont match they go to a tampered state.  Need to decide what to do rules-wise here.  You should probably drop your current messages collection and run in some new ones.  Changed the cancel id as "/" conflicts with URL

CSS builds to a separate file.  I needed it for the unlock (jade) view.  Also makes sense for the single page app as browsers can cache it.

Added a Swift 32A formatter with a simple test

Spun up an SFTP server on msg3dv.  Added some tests to show how to put/get

Got SSL working with the socket stuff.  Have pushed the latest version to the DEV box to see real messages coming in (using loop to push).  http://deck.dev.ims.uss.co.uk

### CODE REVIEW

File: common/formatters/swift/202.js [hunk 1]
```diff
@@ -1,3 +1,28 @@
-export function swift_203(message, args) {
-	return "SWIFT 202 Message TODO: implement formatter";
+import mark from 'markup-js'
+import { swift_field_32A } from './fields'
+
+//"General Financial Institution Transfer"
+export default function swift_202(message, args) {
```
Comment (nikk1e) [inline common/formatters/swift/202.js]: Not working code obvs

**Comment by mavxg** [comment]
I think require('babel-register'); should probably go in server.js and not in the imported files.

**Comment by mavxg** [comment]
I think we should keep log files, server specific config files, and key outside the app directory. The key directory would need to be locked down and that makes deployment of the surrounding app directories more tricky.

**Comment by nikk1e** [comment]
Agree with both points.  I didn't bother looking into where the babel header needed to live...

**Pull request merged**